lua/luasnip/extras/init.lua

```diff
@@ -116,11 +116,18 @@ end
 return {
 	lambda = lambda,
 	match = match,
-	-- repeat a node.
-	rep = function(node_indx)
+	--- Repeat a node, by inserting the text of the passed node.
+	---
+	--- ```lua
+	--- s("extras4", { i(1), t { "", "" }, extras.rep(1) })
+	--- ```
+	---
+	---@param node_ref LuaSnip.NodeRef a single [Node Reference](#node-reference).
```

Comment:
Ah, you can put the relative path to the `DOC.md` in here, so the link also works when viewing the file ;)
(`[Node Reference](../../../DOC.md#node-reference)`, +- a `..` :D)

Reply:
Done in 02cabf1378f3d49146e4add6ddfaa8d0324ea2fb